src/index.js

```diff
@@ -460,21 +460,15 @@ class Superlogin extends EventEmitter {
 		return this._http.get(this._config.baseUrl + 'validate-username/' + encodeURIComponent(username))
 			.then(() => true)
 			.catch(function (err) {
-				if (err.status === 409) {
-					return Promise.reject(false);
-				}
-				throw err.data;
+				return Promise.reject(err);
```

Comment:
You should probably use `throw err.data;` instead;
(It was my bad to use `Promise.reject` in the first place though)